toolkit/scripts/pkggen.mk

```diff
@@ -38,6 +38,7 @@ cached_file              = $(PKGBUILD_DIR)/cached_graph.dot
 preprocessed_file        = $(PKGBUILD_DIR)/preprocessed_graph.dot
 built_file               = $(PKGBUILD_DIR)/built_graph.dot
 output_csv_file          = $(PKGBUILD_DIR)/build_state.csv
+test_results_file		 = $(PKGBUILD_DIR)/test_results.json
```

Comment:
Need tabs to be converted to spaces. In other places too.

Reply:
Thank for catching this, I've fixed this one. Not sure where the others were you referring to?